### PR TITLE
AB#5002

### PR DESCRIPTION
**Pull request by sachin755**
### Proposed changes

*
*

### Related issues

*
*

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by johnjdl** [comment]
Looks good on the local build, @DallasP9124 and/or @r-joyce please review and merge.

**Comment by BigBallard** [comment]
@johnjdl waiting to hear back from Beth, this is not really a bug. Based on the card, these values are expected and are correct even though they both have the vulnerability ID. If I do not get to this by EOD then please ping me to update.

**Comment by BigBallard** [comment]
@johnjdl I am actually confused by the code changes, wasnt he supposed to remove the description and not the name in the RiskObservationLine ?

**Comment by johnjdl** [comment]
@DallasP9124 You are right. That is what the ticket instructed them to do. I'm noticing in the RiskObesrationLine that name was removed and not the description. I will screenshot the code change and send it back.

**Comment by johnjdl** [comment]
@DallasP9124 @r-joyce this has been corrected, please review and merge.

File: opencti-platform/opencti-front/src/private/components/riskAssessment/risks/RiskObservationLine.js [hunk 1]
```diff
@@ -128,7 +128,7 @@ class RiskObservationLineContainer extends Component {
                     variant="h2"
                     style={{ color: 'white' }}
                   >
-                    {observation.node.name && t(observation.node.name)}
```
Comment (BigBallard) [inline opencti-platform/opencti-front/src/private/components/riskAssessment/risks/RiskObservationLine.js]: @AMAN-SHARMA-82591 this needs to be corrected. The name should be rendering, not the description.

**Review by BigBallard — CHANGES_REQUESTED** [review]
@AMAN-SHARMA-82591 I made a note of what needs to be fixed. You removed the name and added the description where the name should have stayed.

File: opencti-platform/opencti-front/src/private/components/riskAssessment/risks/RiskObservationLine.js (hunk 1 above)
Comment (sachin755) [reply opencti-platform/opencti-front/src/private/components/riskAssessment/risks/RiskObservationLine.js]: @DallasP9124  The page you've address is RiskObservationLine which as we've talked in today's meeting will only have name and created-date populated which we have added. If I add another name in that component there will be two fields with same value in it. Is that what you want?

**Review by r-joyce — APPROVED** [review]

**Comment by johnjdl** [comment]
@DallasP9124 @r-joyce please hold off on merging this one.

**Pull request merged**